lighthouse/tests/validator_manager.rs

```diff
@@ -233,9 +249,11 @@ pub fn validator_import_missing_token() {
 }
 
 #[test]
-pub fn validator_import_missing_validators_file() {
+pub fn validator_import_using_both_file_flags() {
     CommandLineTest::validators_import()
         .flag("--vc-token", Some("./token.json"))
+        .flag("--validators-file", Some("./vals.json"))
+        .flag("--keystore-file", Some("./keystore.json"))
```

Comment:
This test is failing: https://github.com/sigp/lighthouse/actions/runs/11404357702/job/31733366200

I found that this is because the `.required(true)` for the `validators-file` flag is removed: https://github.com/sigp/lighthouse/pull/6261/files#diff-e54fe894fb02bdd0c2175fbb4d7cc6d04fa7c8b460b30e9512edb814f569f97eL41

(a it now requires either `--validators-file` or `--keystore-file`)

The test caught it because now that the hard requirement for `--validators-file` is dropped, so missing the validators file flag will not return a `false` from the test. I change this test to testing that if both flags are used, then the test fails. 

Not sure if this is good?

Reply:
As discussed offline, this is good now!